corehq/apps/locations/views.py

```diff
@@ -663,6 +664,8 @@ def location_importer_job_poll(request, domain, download_id, template="hqwebapp/
 
 @locations_access_required
 def location_export(request, domain):
+    if not LocationType.objects.filter(domain=domain).exists():
```

Comment:
i realize people shouldn't be seeing this, but it might be nicer to do something like redirect to the location types page and use django messages to say "you must setup location types before bulk importing" or something